src/client/delphi_epidata.py

```diff
@@ -54,6 +52,21 @@ def log(evt, **kwargs):
         kwargs['timestamp'] = time.strftime("%Y-%m-%d %H:%M:%S %z")
         return sys.stderr.write(str(kwargs) + "\n")
 
+    # One-time check to verify the client version is up to date.
+    @staticmethod
+    def _version_check():
+        try:
+            latest_version = requests.get('https://pypi.org/pypi/delphi-epidata/json').json()['info']['version']
+            if latest_version != __version__:
+                Epidata.log(
+                    "Client version not up to date",
+                    client_version=__version__,
+                    latest_version=latest_version
+                )
+        except Exception as e:
+            Epidata.log("Error getting latest client version", exception=str(e))
+    _version_check.__func__() # run this once on module load
```

Comment:
In Python <= 3.9, static methods cannot be directly called within the class body.
In Python >= 3.10, it works as you'd expect: `_version_check()`

Reply:
```suggestion
    # Run this once on module load. Using dunder method for Python <= 3.9 compatibility 
    # https://stackoverflow.com/a/12718272
    _version_check.__func__()
```

Reply:
plz also add newline above function call!

would it be any better/easier to avoid this pattern here and just call `Epidata._version_check()` at the end of the file (after the class definition)?